api/v1alpha1/openapi.yaml

```diff
@@ -1797,8 +1797,9 @@ components:
     DeviceStatus:
       type: object
       properties:
-        updatedAt:
+        lastSeenAt:
```

Comment:
can we update in this PR: https://github.com/flightctl/flightctl/blob/main/docs/enhancements/fep-000-api-device-fleet.md

https://github.com/flightctl/flightctl/blob/main/docs/enhancements/fep-000-api-device-fleet.md#devicesname-user-facing-method on the yaml example

and then

https://github.com/flightctl/flightctl/blob/main/docs/enhancements/fep-000-api-device-fleet.md#devicesnamestatus-device-side

Where it talks about the heartbeat mechanism?